src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 101;
+    private Ball ball = new Ball();
+
+    public Ball getRandomBall() {
+        ball.setColor();
+        ball.setNumber(new Random().nextInt(BOUND));
+        return ball;
```

Comment:
The 'setColor()' method is being called on 'ball' without any arguments. This method should be provided with an actual color value, likely obtained from a 'ColorSupplier'.